src/parser.ts

```diff
@@ -2125,7 +2146,7 @@ export class Parser {
         return this.finalize(node, new Node.Property('init', key, computed, value, method, shorthand));
     }
 
-    parseRestProperty(params, kind): Node.RestElement {
+    parseRestProperty(params): Node.RestElement {
```

Comment:
Removed the unused parameter as style checks gave warning and couldn't see a reason to keep